go/arrow/array/array_test.go

```diff
@@ -85,10 +85,16 @@ func TestMakeFromData(t *testing.T) {
 		}},
 		{name: "duration", d: &testDataType{arrow.DURATION}},
 
+		{name: "map", d: &testDataType{arrow.MAP}, child: []*array.Data{
+			array.NewData(&testDataType{arrow.STRUCT}, 0, make([]*memory.Buffer, 4), []*array.Data{
```

Comment:
nit: this is probably consistent with other code here, but literal comments like

`/*elementByteWidth=*/4` could make this more readable.

Reply:
added comments for literals